src/encord_active/frontend/src/components/explorer/Explorer.tsx

```diff
@@ -131,14 +168,30 @@ export function Explorer({
     setShowAnnotations(selectedMetric.domain === "annotation");
   }, [selectedMetric.domain, setShowAnnotations]);
 
+  // Data or Label selection
+  const analysisDomainOptions = useMemo(
```

Comment:
This could be a global constant (use strict typing for completeness instead of generation via a mapping).